diff -ruNx '*~' libgcrypt-1.5.0/configure.ac libgcrypt-1.5.0-adding-pc/configure.ac
--- libgcrypt-1.5.0/configure.ac	2011-06-29 10:53:22.000000000 +0200
+++ libgcrypt-1.5.0-adding-pc/configure.ac	2011-12-13 09:19:52.090672599 +0100
@@ -1203,6 +1203,7 @@
 src/gcrypt.h
 src/libgcrypt-config
 src/versioninfo.rc
+src/libgcrypt.pc
 tests/Makefile
 ])
 AC_OUTPUT
diff -ruNx '*~' libgcrypt-1.5.0/src/libgcrypt.pc.in libgcrypt-1.5.0-adding-pc/src/libgcrypt.pc.in
--- libgcrypt-1.5.0/src/libgcrypt.pc.in	1970-01-01 01:00:00.000000000 +0100
+++ libgcrypt-1.5.0-adding-pc/src/libgcrypt.pc.in	2011-12-13 09:20:12.465510447 +0100
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${exec_prefix}/include
+
+Name: libgcrypt
+Description: General purpose crypto library
+Version: @VERSION@
+Requires: libgpg-error
+Libs: -L${libdir} -lgcrypt
+Cflags: -I${includedir} 
diff -ruNx '*~' libgcrypt-1.5.0/src/Makefile.am libgcrypt-1.5.0-adding-pc/src/Makefile.am
--- libgcrypt-1.5.0/src/Makefile.am	2011-05-23 10:07:51.000000000 +0200
+++ libgcrypt-1.5.0-adding-pc/src/Makefile.am	2011-12-13 09:33:02.056375996 +0100
@@ -20,8 +20,12 @@
 
 ## Process this file with automake to produce Makefile.in
 
+pcfiles = libgcrypt.pc
+pkgconfig_DATA = $(pcfiles)
+pkgconfigdir = $(libdir)/pkgconfig
+
 EXTRA_DIST = Manifest libgcrypt-config.in libgcrypt.m4 libgcrypt.vers \
-             gcrypt.h.in libgcrypt.def
+             gcrypt.h.in libgcrypt.def libgcrypt.pc.in
 
 bin_SCRIPTS = libgcrypt-config
 m4datadir = $(datadir)/aclocal
