diff -up libgcrypt-1.8.5/cipher/poly1305-armv7-neon.S.build libgcrypt-1.8.5/cipher/poly1305-armv7-neon.S
--- libgcrypt-1.8.5/cipher/poly1305-armv7-neon.S.build	2017-11-23 19:16:58.000000000 +0100
+++ libgcrypt-1.8.5/cipher/poly1305-armv7-neon.S	2020-01-30 17:26:12.026404286 +0100
@@ -87,9 +87,8 @@ _gcry_poly1305_armv7_neon_init_ext:
 .Lpoly1305_init_ext_neon_local:
 	stmfd sp!, {r4-r11, lr}
 	sub sp, sp, #32
-	mov r14, r2
+	mov r14, #-1
 	and r2, r2, r2
-	moveq r14, #-1
 	UNALIGNED_LDMIA4(r1, r2, r3, r4, r5)
 	GET_DATA_POINTER(r7,.Lpoly1305_init_constants_neon,r8)
 	mov r6, r2
